app/server/main.js

```diff
@@ -2,15 +2,43 @@
 
 exports.start = () => {
 
-    var express = require('express'),
+    // HACKING IN CONFIG OBJECT HERE
+    var config = {
+        server: {
+            port: '3000',
+            api_prefix: '/api/v1',
+            hostname: 'localhost'
+        },
+        github: {
+            clientID: process.env.GITHUB_CLIENTID,
+            clientSecret: process.env.GITHUB_CLIENT_KEY,
+            authRoute: '/auth/github',
+            authCallbackRoute: '/auth/github/callback',
+            failureCallback: '/auth/failure'
+        },
+        session: {
+            secret: 'keyboard cat',
```

Comment:
Is this useful for anyone forking the app, or should it be pulled out as an env variable?